packages/starknet-snap/src/index.test.tsx

```diff
@@ -53,6 +60,19 @@ describe('onRpcRequest', () => {
     ).rejects.toThrow(MethodNotFoundError);
   });
 
+  it('requests gets executed if MetaMask does not needs update', async () => {
+    createMockSpy();
+    expect(
+      await onRpcRequest({
+        ...createMockRequest(),
+        request: {
+          ...createMockRequest().request,
+          method: 'ping',
+        },
+      }),
+    ).toBe('pong');
+  });
```

Comment:
we dont need this test, but i think it is better to add back onInstall and onUpdate test
```suggestion
```